app/models/web_hook_events_daily_aggregate.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class WebHookEventsDailyAggregate < ActiveRecord::Base
+  belongs_to :web_hook
+
+  default_scope { order("created_at DESC") }
+  after_create :aggregate
```

Comment:
I think you could do this `before_create` and then you can avoid 2 calls to the DB